orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/AccountIntervention.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.di.orchestration.shared.entity;
+
+public class AccountIntervention {
+
+    private final AccountInterventionDetails details;
+    private final AccountInterventionState state;
+    private final AccountInterventionStatus status;
+    private final boolean ignoreResetPassword;
+
+    public AccountIntervention(AccountInterventionState state) {
+        this.details =
+                new AccountInterventionDetails(
+                        Long.MIN_VALUE,
```

Comment:
Would `0L` make more sense here? Feels weird to have negative time

Reply:
I thought about that and then decided on `MIN_VALUE` as it's a bit safer.  Using `MIN_VALUE` means `ignorePasswordReset` will never be `true`